mex/extractors/consent_mailer/settings.py

```diff
@@ -7,6 +7,7 @@
 class ConsentMailerSettings(BaseModel):
     """Settings definition class for the consent mailer."""
 
+    request_filter_chunck_size: int = 1
```

Comment:
Plz add pydantic Field to ensure default and description. Maybe consider renaming to something like "backend_fetch_chunk_size" to make it more clear it used against the backend.

```suggestion
    backend_fetch_chunk_size: int = Field(
        1,
        description="The chunk size for fetching against the backend.",
    )
```